doc/internal/man3/ossl_namemap_new.pod

```diff
@@ -55,25 +58,30 @@ names already associated with that number.
 ossl_namemap_name2num() finds the number corresponding to the given
 I<name>.
 
-ossl_namemap_add_n() and ossl_namemap_name2num_n() do the same thing
-as ossl_namemap_add() and ossl_namemap_name2num(), but take a string
+ossl_namemap_add_name_n() and ossl_namemap_name2num_n() do the same thing
+as ossl_namemap_add_name() and ossl_namemap_name2num(), but take a string
 length I<name_len> as well, allowing the caller to use a fragment of
 a string as a name.
 
-
 ossl_namemap_doall_names() walks through all names associated with
 I<number> in the given I<namemap> and calls the function I<fn> for
 each of them.
 I<fn> is also passed the I<data> argument, which allows any caller to
 pass extra data for that function to use.
 
+ossl_namemap_add_names() divides up a set of names given in I<names>,
+separated by I<separator>, and adds each to the I<namemap>, all with
+the same number.  If some of them already exist in the I<namemap>,
```

Comment:
I probably would have used identifier instead of number throughout the doc.

Reply:
This was chosen so we wouldn't confuse them with (constant) NIDs